WordPress/src/main/java/org/wordpress/android/ui/RequestCodes.java

```diff
@@ -14,4 +14,10 @@
     public static final int EDIT_POST       = 800;
     public static final int CREATE_BLOG     = 900;
     public static final int BLOG_SETTINGS   = 1000;
+
+    // Media
+    public static final int ACTIVITY_REQUEST_CODE_PICTURE_LIBRARY = 2000;
```

Comment:
This is nitpicky, but I think we can drop the `ACTIVITY_REQUEST_CODE_` prefix since the class `RequestCodes` is descriptive enough (ex: `RequestCodes.PICTURE_LIBRARY`).